packages/xrpl/src/models/methods/ammInfo.ts

```diff
@@ -0,0 +1,81 @@
+import { Amount, IssuedCurrencyAmount } from '../common'
+
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+/**
+ * The `amm_info` command retrieves information about an AMM instance.
+ * Returns an {@link AMMInfoResponse}.
+ *
+ * @category Requests
+ */
+export interface AMMInfoRequest extends BaseRequest {
+  command: 'amm_info'
+
+  /**
+   * A hash that uniquely identifies the AMM instance.
+   */
+  amm_id?: string
+
+  /**
+   * Specifies one of the pool assets (XRP or token) of the AMM instance.
+   */
+  asset1?: Amount
+
+  /**
+   * Specifies the other pool asset of the AMM instance.
+   */
+  asset2?: Amount
+}
+
+/**
+ * Response expected from an {@link AMMInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface AMMInfoResponse extends BaseResponse {
+  result: {
+    /**
+     * The account that tracks the balance of LPTokens between the AMM instance via Trustline.
+     */
+    AMMAccount: string
```

Comment:
Why are these capitalized? (In other files the response is also lowercase - e.g. AccountInfoResponse)